packages/hub/indexing/running-indexers.js

```diff
@@ -88,6 +88,15 @@ module.exports = class RunningIndexers {
 
   async update(forceRefresh, hints) {
     await this._loadSchemaModels();
+
+    await Promise.all(Object.keys(this.branches).map((branch) => {
+      return this.branches[this.controllingBranch].add(
+        'branches',
+        branch,
+        { data: {id: branch, type: 'branches', attributes: {}}},
+      );
+    }));
+
```

Comment:
@mansona  this specific chunk of code is what is causing the test failures that you were asking me about.
```
error: database "pgsearch_test_1547739697309" does not exist
```
I don't know why, though--perhaps @ef4 does....